crates/gitbutler-core/src/virtual_branches/virtual.rs

```diff
@@ -1968,7 +1968,11 @@ fn virtual_hunks_into_virtual_files(
             let conflicted =
                 conflicts::is_conflicting(project_repository, Some(&id)).unwrap_or(false);
             let binary = hunks.iter().any(|h| h.binary);
-            let modified_at = hunks.iter().map(|h| h.modified_at).max().unwrap_or(0);
+            let modified_at = hunks
+                .iter()
+                .map(|h| h.modified_at)
+                .max()
+                .unwrap_or(Duration::ZERO);
```

Comment:
I like this constant - typically I end up with `Duration::default()`, but this is more expressive and works in `const` contexts.